### PR TITLE
issue: 822562 Fix segmentation fault in vma_stats while using long

### PR DESCRIPTION
**Pull request by liranoz12**
sockets range.

Signed-off-by: Liran Oz <lirano@mellanox.com>

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/216/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/202/ for details (Mellanox internal link).

**Comment by igor-ivanov** [comment]
bot:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/204/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/218/ for details (Mellanox internal link).

**Comment by igor-ivanov** [comment]
bot:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/206/ for details (Mellanox internal link).

File: src/stats/stats_reader.cpp [hunk 1]
```diff
@@ -1183,8 +1183,13 @@ int analize_fds_range(char* range)
 	char* left = range;
 	char* right = NULL;
 	char* delim_loc = NULL;
-	char range_copy[10];
+	char range_copy[101];
```
Comment (rosenbaumalex) [inline src/stats/stats_reader.cpp]: @liranoz12 is there any meaning for 101 byte? or just bigger number than 10?

Comment (liranoz12) [inline src/stats/stats_reader.cpp]: @rosenbaumalex Just bigger number than 10.

**Comment by rosenbaumalex** [comment]
+1

**Pull request merged**